BlargBucket/ViewModels/ReadMeViewModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ReadMeViewModel.swift
+//  BlargBucket
+//
+//  Created by Nick Smillie on 12/23/14.
+//  Copyright (c) 2014 Luke Deniston. All rights reserved.
+//
+
+import Foundation
+
+class ReadMeViewModel: NSObject {
+	
+	dynamic var readMeMarkDown: String?
+	var readMeAttributed: NSAttributedString?
```

Comment:
I think it would be better to drop the `readMeMarkDown` property, and instead just rely on a single method to set up the attributed string. Something like `parseMarkDown(readMeMarkDown:String?)`. Move all your logic from `setAttributedString` into that new method. You get two things from that:

- You reduce the "surface area" of this class. If this were to be reused, I could see a lame developer (me) getting confused about which property to use (`readMeMarkDown` vs. `readMeAttributed`).
- You avoid having your class be in a "half initialized" state (where `readMeMarkDown` has been set, but `setAttributedString()` hasn't been called yet.